pkg/streamingpromql/functions_test.go

```diff
@@ -50,3 +58,94 @@ func TestRegisterScalarFunctionOperatorFactory(t *testing.T) {
 	// Cleanup changes to instantVectorFunctionOperatorFactories
 	delete(scalarFunctionOperatorFactories, "new_function")
 }
+
+// This test ensures that all functions correctly merge series after dropping the metric name.
+func TestFunctionDeduplicateAndMerge(t *testing.T) {
+	data := `
+		load 30s
```

Comment:
We should use spaces here as tabs don't ensure alignment for all

![image](https://github.com/user-attachments/assets/62b202b7-2473-49d3-8095-fbf5a812a138)

Reply:
🤦 Fixed in https://github.com/grafana/mimir/pull/10553